message-index/messages/GHC-69158/example/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Name clash due to mix of qualified and unqualified re-export
+---
+
+## Error Message
+
+```
+Greeting.hs:3:5: error: [GHC-69158]
+    Conflicting exports for ‘greeting’:
+       ‘W.greeting’ exports ‘W.greeting’
+         imported qualified from ‘World’ at Greeting.hs:6:1-27
+         (and originally defined at World.hs:4:1-8)
+       ‘U.greeting’ exports ‘U.greeting’
+         imported qualified from ‘Universe’ at Greeting.hs:7:1-30
+         (and originally defined at Universe.hs:4:1-8)
+  |
+3 |   , U.greeting
+  |     ^^^^^^^^^^
+```
+
+## Explanation
+
+This examples defines the modules `World` and `Universe`, both of which export
+an identifier named `greeting`. The `Greeting` module then imports `World` and
+`Universe` qualified, meaning it can use both `greeting`s in the body of
+`bothGreetings` without any ambiguity. However, the export of both `greeting`s
```

Comment:
Reading this aloud, I can't distinguish between `bothGreetings` and both `greeting`s, which I tripped over. What about:
```suggestion
`bothGreetings` without any ambiguity. However, the export of both definitions named `greeting`
```

Reply:
Should be fixed now!